IshiHisashi/GlucoFit
# style: JDEV-80 - Create theme config and update button component

## Description of this change
- add theme config containing fonts and colours referring to UI kit https://www.figma.com/design/b475FeQKtPyeB7kL2ciQGQ/UI-Kit?node-id=17-2&t=z5vQJSMSPykIIMar-1
- update button component
- make svg colour customizable

## Steps to test
1. `cd frontend`, `npm install`, and run the app on an emulator
2. You can see the appearance of the buttons on Insights screen. 

<img src="https://github.com/user-attachments/assets/8d7ef52a-a358-4028-abcc-22b2a7eb46ec" width="150px" >


## Ticket link
https://team-jupiter-capstone.atlassian.net/browse/JDEV-80
